physionet-django/console/views.py

```diff
@@ -815,8 +816,15 @@ def process_credential_application(request, application_slug):
     Process a credential application. View details, contact reference,
     and make final decision.
     """
-    application = CredentialApplication.objects.get(slug=application_slug,
-        status=0)
+    try:
+        application = CredentialApplication.objects.get(slug=application_slug,
+            status=0)
+    except ObjectDoesNotExist:
```

Comment:
The model already has a way to handle NotExists with "CredentialApplication.DoesNotExist"

Reply:
Good point. Switched!